pkg/provision/storage/perWorkspaceStorage.go

```diff
@@ -157,9 +159,12 @@ func syncPerWorkspacePVC(workspace *dw.DevWorkspace, clusterAPI sync.ClusterAPI)
 
 	// TODO: Determine the storage size that is needed by iterating through workspace volumes,
 	// adding the sizes specified and figuring out overrides/defaults
-	pvcSize := constants.PVCStorageSize
+	pvcSize := *config.Workspace.DefaultStorageSize.PerWorkspace
 	if namespacedConfig != nil && namespacedConfig.CommonPVCSize != "" {
-		pvcSize = namespacedConfig.CommonPVCSize
+		pvcSize, err = resource.ParseQuantity(namespacedConfig.CommonPVCSize)
```

Comment:
We should maybe drop this whole `namespacedConfig` check, since using the per-namespace `CommonPVCSize` doesn't really make sense here.